app/lib/asyncSuspenseValue.ts

```diff
@@ -32,6 +32,21 @@ export function createAsyncSuspenseValue<T>(getValue: () => Promise<T>) {
   };
 
   const asyncValue = {
+    load: async () => {
```

Comment:
I thought i'd use this in event handlers... but it wasn't convenient so I ended up just adding `.read()` to all of the hooks using this database suspense value. I left this in the PR because it might come useful in the future